modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -27,6 +33,7 @@
           <AposModalTabsBody>
             <div class="apos-doc-editor__body">
               <AposSchema
+                v-if="docReady"
```

Comment:
If we get this up immediately the populated doc isn't ready yet and it's all blank.